packages/features/apps/AdminAppsList.tsx

```diff
@@ -183,26 +185,36 @@ const querySchema = z.object({
 const AdminAppsList = ({
   baseURL,
   className,
+  fromAdmin,
+  currentStep,
+  setIsLoading,
   useQueryParam = false,
 }: {
   baseURL: string;
   className?: string;
+  fromAdmin?: boolean;
+  currentStep?: number;
+  setIsLoading?: Dispatch<SetStateAction<boolean>>;
   useQueryParam?: boolean;
 }) => {
   const router = useRouter();
   return (
     <form
-      id="wizard-step-2"
-      name="wizard-step-2"
+      id={`wizard-step-${currentStep}`}
+      name={`wizard-step-${currentStep}`}
       onSubmit={(e) => {
         e.preventDefault();
+        setIsLoading && setIsLoading(true);
         router.replace("/");
       }}>
       <AppCategoryNavigation
         baseURL={baseURL}
-        fromAdmin
+        fromAdmin={fromAdmin}
```

Comment:
Is this being used elsewhere when fromAdmin is not true?

Reply:
Yeah, `fromAdmin` is false on the /apps page. It's true from the Settings pages.